### DISCUSSION TITLE
Some questions about i3

### DISCUSSION THREAD

**Discussion by pizzuhh** [i3/i3#6009] [Q&A] [answered]
I have used [dwm](https://dwm.suckless.org/) for some time and decided to go back to i3 but there are few things i want to ask if are possible to change from the config files:
1. Can i display all of the work spaces in the default bar? like even if they are not enabled can i display all of them (from 1 to 10)?
2. When i switch work spaces there's something like a transition. Can it be disabled?

**Comment by orestisfl** [comment, marked as answer]
1. i3 doesn't have an "all work spaces" concept, workspace names are dynamic. However, you can programmatically control which workspace buttons are displayed in the i3bar since i3 4.23: https://i3wm.org/docs/i3bar-workspace-protocol.html
2. i3 doesn't have any animations, this is probably a compositor you are using. The most popular compositor to use with i3 is `picom`. Try `killall picom` and see if the transitions are gone. If yes, try configuring picom to disable these transitions.

**Reply by pizzuhh** [reply]
Thanks it was picom. I've changed the `fading` to false
